src/components/outlines/editor/index.js

```diff
@@ -1,27 +1,22 @@
+import React, { PropTypes } from 'react';
 import reactStamp from 'react-stamp';
+import EditorFactory from 'components/editor';
```

Comment:
The implementation and usage of the editor will change with the new version (such as with data fetching). I have separate issues to be created for migrating to the new editor in other places. To keep the tests passing, let's go ahead and leave this file unchanged, and we'll remove it in another issue.